2.7.0/todo/result_onnx.md

```diff
@@ -26,15 +25,23 @@ The categories below are as follows:
 * not user facing: All commits that are not public end-user facing and hence should be dropped from the release notes
 
 ## onnx
+
 ### bc breaking
+
 - [ONNX] Set USE_EXPERIMENTAL_LOGIC to True ([#137296](https://github.com/pytorch/pytorch/pull/137296))
```

Comment:
This needs a description of what is changing and a before/after--please follow the guidelines here: https://docs.google.com/document/d/14OmgGBr1w6gl1VO47GGGdwrIaUNr92DFhQbY_NEk8mQ/edit?tab=t.0#heading=h.a9htwgvvec1m